grails-app/controllers/imod/AssessmentTechniqueController.groovy

```diff
@@ -96,10 +97,12 @@ class AssessmentTechniqueController {
 
 		// This checks when a technique is assigned to a learning objective
 
+
 		if(params.assigncheck == true) {
-							// get current user object
-		def currentLearningObjective = LearningObjective.findById(learningObjectiveID)
+		//println("goes into asigncheck if:: "+params.assigncheck )

```

Comment:
The code inside the `if` block.
between the `{}` should be tabbed one additional level.